test/task.h

```diff
@@ -184,6 +184,7 @@ enum test_status {
 #  define inline __inline
 # endif
 
+# if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
For MSVC 2013, the docs do mention inline, but I get
 ```
test.c(2) : error C2054: expected '(' to follow 'inline'
```
For MSVC 2015 it works.